internal/constants/constants.go

```diff
@@ -177,6 +179,7 @@ const (
 	ConnectorDeploymentModeShared   = "shared"
 	ConnectorDeploymentModeUnshared = "unshared"
 
+	ConnectorStorageClassAnnotation       = "connector.vanus.ai/storage_class"
```

Comment:
I think it's better to use ConnectorWorkloadTypeAnnotation = "connector.vanus.ai/workload_type", which defaults to the Deployment type.
The ConnectorStorageClassAnnotation is still retained but is not required. If it is empty, it is the default storageclass, otherwise it is the specified storageclass.

Reply:
modify